src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -105,8 +107,14 @@ export class SearchInterceptor {
         mergeMapTo(throwError(new RequestTimeoutError()))
       );
 
+      const userAbort$ = fromEvent(this.abortController.signal, 'abort').pipe(
```

Comment:
Remember this possible issue we discussed? https://github.com/elastic/kibana/issues/65051
Would it make sense to take at a look at it here while you on it? 
Feel free to skip, if plan to address that separately

Reply:
Would prefer to address separately - because this PR is just about UI changes :pray: